### PR TITLE
fix: Add a prebuild:dev before doing a build:dev to copy the required files

### PR DESCRIPTION
**Pull request by LoganPB**
## 📝 What is this PR

Add a prebuild:dev to the package.json because when we're doing the build:dev, the manifest and assets are not copied and there's an error on chrome at the importation. 

## 🏭 How does it work

**Briefly explain how you tackled the issue, what were the alternatives**

## 🧪 QA Process

Do a build:dev and import the dev-build folder on chrome extension manager

## 👌 To check

-   [x] There's no regression
-   [x] Coding guidelines are followed ([Contributing](https://github.com/phantombuster/engineering-docs/blob/master/contributing.md), [Testing](https://github.com/phantombuster/engineering-docs/blob/master/testing.md))

## 🚀 What to do after deployment

**Who should be notified after the deployment of this PR? What should be written in the changelog on Slack?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
 		"watch": "npm-run-all -p watch:*",
 		"clean": "rm -rf dist beta dev-build *.zip && cpy manifest.json dist && cpy src/assets/* dist/assets && cpy src/assets/* beta/assets && cpy src/assets/* dev-build/assets",
 		"prebuild": "npm run clean",
+		"prebuild:dev": "npm run clean",
```
Comment (Oxygen66) [inline package.json:16]: How does it work? How is `dev` taken into consideration?

**Review by Oxygen66 — APPROVED** [review]
LGTM 👍  Just one question ^^

File: package.json (hunk 1 above)
Comment (LoganPB) [reply package.json:16]: The pre prefix looks for a similar script with a matching name. When the script is called, the prescript is called automatically. [https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts)

**Review by stressGC — APPROVED** [review]
Code LGTM, I don't know the extension's code though, please wait for another approval

**Review by RiahiKarim — APPROVED** [review]
Nice. LGTM 💯 .

**Pull request merged**